.github/ISSUE_TEMPLATE/bug.yaml

```
@@ -0,0 +1,63 @@
+name: Bug Report
+description: File a bug report.
+title: "[Bug]: "
+type: Bug
+body:
+  - type: markdown
+    attributes:
+      value: |
+        Thanks for taking the time to fill out this bug report!
+  - type: input
+    attributes:
+      label: OS
+      description: |
+        Describe your Operating System. Example:
+          - **OS**: Ubuntu 18.04
+      value: |
+        OS:
```

```suggestion
      placeholder: Ubuntu 24.04
```

this is redundant since this field will already be labelled OS